### DISCUSSION TITLE
Rework In Progress

### DISCUSSION THREAD

**Discussion by Darthsae** [Darthsae/TCreator#8] [Announcements]
The current codebase has been experiencing multiple issues and getting the features needed implemented has been almost impossible. Due to this I have been working on rewriting the project with another developer in c++, so a new update supporting items and npcs, along with support for both TModLoader and the TCreator API developed by Jamz is on the horizon, and due to this distribution of TCreator will be much simpler than previously. However due to this it will be a couple months until it releases. Thank you for your patience.

**Comment by AwesomeSavage** [comment]
Good to know.
